sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -1375,6 +1376,7 @@ public partial class LocationExpanded
     public partial class LocationMetadata
     {
         internal LocationMetadata() { }
+        public string Geography { get { throw null; } }
```

Comment:
there is a new readonly property in the latest swagger actually.
we never have it because previously we are suppressing the generated version of this model.